Description: Support bulds on older aufs
 Under aufs on squeeze, tcgetattr can return EINVAL instead of ENOTTY
 See https://lsbbugs.linuxfoundation.org/show_bug.cgi?id=2771
Forwarded: not-needed
Author: Stefano Rivera <stefanor@debian.org>
Last-Update: 2013-02-23

--- a/pypy/module/termios/test/test_termios.py
+++ b/pypy/module/termios/test/test_termios.py
@@ -143,7 +143,7 @@
         fd = os.open('.', 0)
         try:
             exc = raises(termios.error, termios.tcgetattr, fd)
-            assert exc.value.args[0] == errno.ENOTTY
+            assert exc.value.args[0] in (errno.ENOTTY, errno.EINVAL)
         finally:
             os.close(fd)
 
